app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>This is the Users#index page.</h1>
+<% @users.each do |user| %>
+  <div class="user_info">
+   <%= image_tag ("#{user.photo}"), alt: "#{user.name}'s photo", width: "100px", height: "80px" %>
+   <span class="text_info">
+   <span class="user_name"><%= "Name: #{user.name}"%></span>
+   <br>
```

Comment:
- [ ] I would suggest making the name clickable which takes the user to the user's show page.

Reply:
Thank you for the suggestion. I have worked on it.